coalib/results/Diff.py

```diff
@@ -20,18 +20,26 @@ def __init__(self, file_list=None,
         Creates an empty diff for the given file.
 
         :param file_list: The original (unmodified) file as a list of its
-                          lines.
+                          lines. It has to be provided if ``create`` is not set,
+                          else an error will be raised.
         :param rename:    False or str containing new name of file.
         :param delete:    True if file is set to be deleted.
         :param create:    False or str containing the file name of the file to
                           be created.
+        :raises ValueError: If ``file_list`` is not provided and ``create`` is
+                            not set.
         """
         self._changes = {}
-        self._file = [] if create else file_list
```

Comment:
You just added this line in the prev commit, now you're replacing it with the older line and copying it at L41. Couldn't understand why ^^